src/handlers/motions/motionStaked/helpers.ts

```diff
@@ -241,10 +249,9 @@ export const getUpdatedMessages = ({
     });
   }
 
-  // Only send am ObjectionFullyStaked message if the motion has not already been fully staked for the YAY side
+  // Only send an ObjectionFullyStaked message if the motion has not already been fully staked for the YAY side
```

Comment:
I like it when you correct my (most likely anyway) spelling mistakes in comments :)